packages/abstractions/src/requestInformation.ts

```diff
@@ -131,7 +131,14 @@ export class RequestInformation {
    */
   public setQueryStringParametersFromRawObject = (q: object): void => {
     Object.entries(q).forEach(([k, v]) => {
-      this.queryParameters[k] = v;
+      let key = k;
+      if ((q as any).getQueryParameter) {
+        const serializationKey = (q as any).getQueryParameter(key) as string;
```

Comment:
Please add  an integrated test  in packages/test which passes query params and verify if the real time tests work with query params

Reply:
Add a test  to verify https://graph.microsoft.com/v1.0/users?$search=%22displayName:%2523%22&$count=true